src/Security/Authentication/Negotiate/src/Internal/NegotiateLoggingExtensions.cs

```diff
@@ -60,6 +61,10 @@ static NegotiateLoggingExtensions()
                 eventId: new EventId(10, "ClientError"),
                 logLevel: LogLevel.Debug,
                 formatString: "The users authentication request was invalid.");
+            _negotiateStateErrorType = LoggerMessage.Define<string>(
+                eventId: new EventId(11, "NegotiateStateErrorType"),
+                logLevel: LogLevel.Information,
+                formatString: "Negotiate State BlobErrorType of {blobErrorType}.");
```

Comment:
This is an awkward error message. It doesn't seem meaningful the the reader.
```suggestion
                formatString: "Negotiate error code: {error}.");
```